packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/base-load-balancer.ts

```diff
@@ -127,9 +129,7 @@ export abstract class BaseLoadBalancer extends Resource {
 
     const internetFacing = ifUndefined(baseProps.internetFacing, false);
 
-    const vpcSubnets = ifUndefined(baseProps.vpcSubnets,
-      { subnetType: internetFacing ? ec2.SubnetType.PUBLIC : ec2.SubnetType.PRIVATE });
-
+    const vpcSubnets = (internetFacing ? { subnetType: ec2.SubnetType.PUBLIC } : {});
```

Comment:
Didn't you just remove using `props.vpcSubnets` if supplied? This shouldn't be passing tests if true...

Reply:
Oho. It turns out that none of the tests for elbv2 actually supply vpcSubnets, which is why the test is passing even though I removed that. I'll fix my code and add proper tests.